Add `android.intent.action.TTS_SERVICE` to manifest file

The Text-to-Speech engine is not loading on Oppo devices (running ColorOS), and the solution might be to follow the instructions at https://developer.android.com/reference/android/speech/tts/TextToSpeech

```java
2025-03-04 20:45:57.954 17212-17212 ai.elimu.v...lication$1 ai.elimu.vitabu.debug                I  status: -1
2025-03-04 20:45:57.954 17212-17212 AndroidRuntime          ai.elimu.vitabu.debug                D  Shutting down VM
2025-03-04 20:45:57.955 17212-17212 AndroidRuntime          ai.elimu.vitabu.debug                E  FATAL EXCEPTION: main
                                                                                                    Process: ai.elimu.vitabu.debug, PID: 17212
                                                                                                    java.lang.RuntimeException: Unable to create application ai.elimu.vitabu.BaseApplication: java.lang.NullPointerException: Attempt to invoke virtual method 'int android.speech.tts.TextToSpeech.setSpeechRate(float)' on a null object reference
                                                                                                    	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:8215)
                                                                                                    	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2642)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:107)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:282)
                                                                                                    	at android.os.Looper.loop(Looper.java:387)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:9494)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:600)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1005)
                                                                                                    Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int android.speech.tts.TextToSpeech.setSpeechRate(float)' on a null object reference
                                                                                                    	at ai.elimu.vitabu.BaseApplication$1.onInit(BaseApplication.java:35)
```

---

* Relates to #111

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -10,6 +10,9 @@
 
     <queries>
         <provider android:authorities="${contentProviderApplicationId}.provider.storybook_provider" />
+        <intent>
+            <action android:name="android.intent.action.TTS_SERVICE" />
+        </intent>
     </queries>
 
     <application

